tools/gcp/create

```diff
@@ -1,6 +1,13 @@
 #!/bin/sh
 
-gcloud compute instances create \
-    $@ \
-    --image tfe-image \
-    --machine-type=n1-standard-4
\ No newline at end of file
+for instance in $@
+do
+    gcloud compute instances create-with-container $instance \
+        --container-image=docker.io/tfencrypted/tf-encrypted \
+        --container-command=python \
+        --container-arg="tf_encrypted/player/__main__.py" \
```

Comment:
why not `-m tf_encrypted.player`?

Reply:
Might be another way to fix this but the command being run in the docker container wasn't being parsed correctly and causing the command to be chopped up into separate and not work. Think this will do for now.